File: twosum.py
class Solution(object):
    def twoSum(self, nums, target):
        solution = []
        for i in range(len(nums)):
            for j in range(i+1, len(nums)):
                if (nums[i] + nums[j] == target):
                    solution.append(i)
                    solution.append(j)
                    break
                else:
                    continue

        return solution

File: TwoSum.cs
public class Solution {
    public int[] TwoSum(int[] nums, int target) {
        List<int> answer = new List<int>();
        for (int i = 0; i < nums.Length; i++){
            for (int j = i + 1; j < nums.Length; j++){
                if (nums[i] + nums[j] == target){
                    answer.Add(i);
                    answer.Add(j);
                }
            }
        }
       int[] twosum =  answer.ToArray();
       return twosum;
    }
}